templates/docs/examples/patterns/search-and-filter/chip-overflow.html

```diff
@@ -56,7 +55,7 @@
   </div>
   <div class="p-search-and-filter__panel" aria-hidden="true">
     <div class="p-filter-panel-section">
-      <h3 class="p-filter-panel-section__heading">Cloud</h3>
+      <h5 class="p-filter-panel-section__heading">Cloud</h5>
```

Comment:
Would be good to make sure this change is reflected in the react component (not sure if the heading level is hardcoded there).
Or even better, if it's a styling change, it should happen in CSS `p-filter-panel-section__heading` rather than via HTML.

Reply:
This is done in HTML and CSS [here](https://github.com/canonical/vanilla-framework/blob/f0e4bd851b7c6bb98a6401a06ecad38b4345293f/scss/_patterns_search-and-filter.scss#L170-L173)

I wanted to change it in both places so that the markup is still semantically clear. However, using h3 with the updated styling has no visual differences from just using h5.

That being said, the markup change should be added to React Components for completeness.